spec/spec_helper.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../book'
+require_relative '../classroom'
+require_relative '../decorate'
+require_relative '../person_class'
+require_relative '../rental'
+require_relative '../student_class'
+require_relative '../teacher_class'
+
+describe Book do
+  before :each do
+    @book = Book.new('All girls are the same', 'Tobby')
+  end
+
+  describe '#addnew' do
+    it 'takes the two parameter' do
+      expect(@book).to be_an_instance_of Book
+    end
+  end
+
+  describe 'check value of title and author' do
+    it 'value of title' do
+      expect(@book.title).to eq('All girls are the same')
+    end
+
+    it 'value of author' do
+      expect(@book.author).to eq('Tobby')
+    end
+  end
+end
+
+describe 'decorator' do
+  before :each do
+    @person = Person.new(22, 'oluwatobiloba')
```

Comment:
- [ ] I think it would be great if you test namable clas as well since it is required to all classes that you created while implementing the Decorator design pattern and you created Nameable class so i think it would be great if you test the method inside it `correct_name` 🚀💯✅🔥💪🏻

![image](https://user-images.githubusercontent.com/58553711/233384708-9e9e17b0-dc7f-4ae6-8d36-2729ad649ee6.png)